pkg/wifi/iwl.go

```diff
@@ -169,15 +169,14 @@ func (w *IWLWorker) Connect(stdout, stderr io.Writer, a ...string) error {
 	}
 
 	// Each request has a 30 second window to make a connection
-	ctx, cancel := context.WithTimeout(context.Background(), 200*time.Second)
+	ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
 	defer cancel()
 	c := make(chan error, 1)
 
 	// There's no telling how long the supplicant will take, but on the other hand,
 	// it's been almost instantaneous. But, further, it needs to keep running.
 	go func() {
-		cmd := exec.CommandContext(ctx, "/usr/bin/strace", "-f", "-o", "/tmp/out", "-s", "1024", "-v", "wpa_supplicant", "-i"+w.Interface, "-c/tmp/wifi.conf", "-dd")
-
+		cmd := exec.Command("wpa_supplicant", "-i"+w.Interface, "-c/tmp/wifi.conf")
 		outfile, err := os.OpenFile("logOutput.txt", os.O_APPEND|os.O_CREATE|os.O_WRONLY, 0644)
 		cmd.Stdout, cmd.Stderr = outfile, outfile
 		if err != nil {
```

Comment:
On line 193, I think this should also push an error to the channel. Since if wpa_supplicant exits for any reason, it is an error.